### DISCUSSION TITLE
Support for account with multiple currencies

### DISCUSSION THREAD

**Discussion by KarelBeyr** [ghostfolio/ghostfolio#1737] [Ideas]
I come from non-US market, and in my portfolio I have stocks that are traded (sorry I don't know the correct buzzword) in different currencies. Some in US, some in EUR, some in CZK. Am I supposed to create multiple accounts (even if it's still the same broker account in real life) for each currency?
This leads me to a bug - some stocks are traded in one currency, but they pay dividends in different currency (e.g. ERBAG.PR - traded in CZK, pays dividends in EUR) - this behaviour is connected to previous paragraph, but currently is completely broken - as Ghostfolio correctly gets info about dividend amount, but unfortunately does not convert EUR to CZK.
Perhaps with some architectonic guidance I may be able to help fix both of those behaviours.

**Comment by KarelBeyr** [comment]
Ultimately I solved it in Ghostfolio by having multiple accounts: one for each currency in each broker.
And dividends in different currency than the stock - I solved this by transforming the dividend history before importing to Ghostfolio (so I do not use "automatic dividend import" feature for such dividends).

**Reply by Marx2** [reply]
Do you need to split your import, so separate CSV file for each currency=account?
